modules/currency/api/currency-routes.ts

```diff
@@ -5,8 +5,48 @@ import GBP_USD_TICKER_DATA from '../data/gbp-usd-ticker-data';
 import { getQtyFromRequest } from '../../../utils/route-utils';
 import getDigitalCurrencyAddress from '../utils/getDigitalCurrencyAddress';
 import getDigitalCurrencyTxList from '../utils/getDigitalCurrencyTxList';
+import getDigitalCurrencyLedger from '../utils/getDigitalCurrencyLedger'
 import DigitalCoinEnum from '../consts/DigitalCoinEnum';
 
+
+/**
+ * @openapi
```

Comment:
Hi, can you move this definition comment into the LedgerInfo.ts please :)

Reply:
all set :)